### PR TITLE
Example and links added for override layout section

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) updates the example to the _override layout_ section and provided the links to the override the layout page.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/themes/theme-inherit.html

### CODE REVIEW

File: src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md [hunk 1]
```diff
@@ -169,3 +169,27 @@ To override the instructions from an ancestor theme layout file:
 To override module [layout instructions](https://glossary.magento.com/layout-instructions) (base layout):
 
 *  Create a layout file with the same name in the `<theme_dir>/<Vendor>_<Module>/layout/override/base` directory.
+
+**Example:**
+
+OrangeCo decided they should modify the product page layout completely to meet the new design, defined in `<Magento_Catalog_module_dir>/view/frontend/layout/catalog_product_view.xml`
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:175]: My only comment here is that OrangeCo exists. Suggest using a name that doesn't?

**Review by BarnyShergold — APPROVED** [review]
Approved but there is comment about the company name in the example

File: src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md (hunk 1 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:175]: @BarnyShergold  We need to use an imaginary company name that doesn't exist. Right?
If so, can we change the OrangeCo to something like _SampleCompany?_

Comment (BarnyShergold) [reply src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md:175]: That would work @ajithkumar-maragathavel

Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md]: @BarnyShergold I have updated the company name. Please review it.

**Review by BarnyShergold — APPROVED** [review]
Great stuff!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**